mis_builder/views/mis_report.xml

```diff
@@ -102,12 +103,13 @@
             </form>
         </field>
     </record>
+
     <record id="mis_report_view_kpi_form" model="ir.ui.view">
         <field name="name">mis.report.view.kpi.form</field>
         <field name="model">mis.report.kpi</field>
         <field name="arch" type="xml">
             <form string="MIS Report KPI">
-                <group col="4">
```

Comment:
A little doubt about this one. Can you show the before / after screenshot?

Reply:
- Before with `col="4"`

![image](https://user-images.githubusercontent.com/46039081/219267167-bea0fd3c-1105-44f7-adfb-bb476419da98.png)

- After with `colspan="4"`

![image](https://user-images.githubusercontent.com/46039081/219267422-61867a18-2ff4-402d-8527-e36e35303aa0.png)

The main diff between them are about the right padding of the fields, indeed it should not have been included in the PR in order to fascilate the review

Reply:
This looks better indeed. If this applies to other versions, don't hesitate to do a PR to any version, and I'll forward/backport.